### ISSUE TITLE
Frequent performance issues

### ISSUE THREAD

**Issue by dabinat** [mozilla/voice-web#1801]
Over the past couple of days I have experienced the following issues on the site:

* Long page loading times
* The validation page appearing blank for 5-10 secs before everything appears
* The validation page appearing blank indefinitely
* Fewer than 5 clips appearing on the validation page
* Some clips having completely blank text
* Messages saying there are no more clips to validate (this is obviously false because English has a big backlog)
* Occasionally a 503 Service Unavailable error
* Occasionally the message “An error occurred. Sorry!”

**Comment by Gregoor** [comment]
Thanks for the report and sorry for the complications! The servers were updated more than usual last week, which led to more cache rewarming (which I think was the cause of those issues).

RE the 503 & An error occured: I assume this was on the dashboard page? I have a fix for that in the pipeline.

**Comment by dabinat** [comment]
It happened on multiple pages, but I believe it was most common on the Dashboard.

**Comment by dabinat** [comment]
I haven’t been able to visit the site at all today. I get “An error occurred. Sorry!” every time.

**Comment by dabinat** [comment]
This does seem to be cache related. I was able to use the site on my desktop computer (which resets whenever the browser is closed) but I couldn’t use it on my phone until I cleared my browser cache.

So I guess this is related to my browser caching old scripts? On my own site I give scripts dummy version numbers (e.g. script.js?v=1234) and then just increment the number whenever there’s a script change. That way it forces the browser to download the new version and not use the old cached version.
